python/vast/utils/arrow.py

```diff
@@ -11,3 +13,25 @@ def names(schema: pa.Schema):
 def name(schema: pa.Schema):
     xs = names(schema)
     return xs[0] if xs[0] else ""
+
+
+# Accepts a 128-bit buffer holding an IPv6 address and
+# returns an IPv4 or IPv6 address.
+def unpack_ip(buffer: SupportsBytes) -> str:
+    num = int.from_bytes(buffer, byteorder='big')
+    # convert IPv4 mapped addresses back to regular IPv4
+    # https://tools.ietf.org/html/rfc4291#section-2.5.5.2
+    if (num >> 32) == 65535:
+        num = num - (65535 << 32)
+    return ipaddress.ip_address(num)
+
+
+def fmt_ip(df: pd.DataFrame, ip_columns: list[str]) -> pd.DataFrame:
```

Comment:
Open question: should there be a default for the common Suricata and Zeek fields, like `['src_ip', 'dest_ip', 'id.orig_h', 'id.resp_h']`?

Reply:
The ultimate answer would be [OCSF](https://schema.ocsf.io).

The pragmatic answer would be: define a VAST model in terms of concepts and use that as layout, and convert that here manually.

This shows the need for the `taxonomize` pipeline operator.

Reply:
Thanks for the leads. So I guess for now it's fine to leave as is (without a default value)? (If so feel free to resolve).